Code/Tools/SceneAPI/SceneData/GraphData/MeshData.cpp

```diff
@@ -187,26 +187,8 @@ namespace AZ
             void MeshData::GetDebugOutput(SceneAPI::Utilities::DebugOutput& output) const
             {
                 output.Write("Positions", m_positions);
-                int index = 0;
```

Comment:
Removing this makes sense. This output ended up being too noisy to be useful. Right now anyone who needs to see data this deep can locally update the debug output for the scene graph with other info they need. I'm just adding this comment in case anyone else has questions about this code removal.